02-electric/electric-section-two/electric-section-two.css

```diff
@@ -1,88 +1,345 @@
+  justify-content: center;
+  row-gap: 30px;
+  padding: var(--subcontainer-padding-mobile);
   border: solid 1px black;
+  background-color: var(--level1-background-color);
+}
+
+@media (min-width:500px){
+  .electric-section-two-subcontainer{
+    padding: var(--subcontainer-padding-desktop);
+  }
 }
 
 .electric-section-two-title {
   text-align: center;
-  font-size: var(--title-size-mobile);
+  font-size: var(--title-font-size-mobile);
+  font-weight: var(--title-font-weight-mobile);
 }
 
-/*******PART ONE*****/
+@media (min-width: 500px) {
+  .electric-section-two-title {
+    font-size: var(--title-font-size-desktop);
+    font-weight: var(--title-font-weight-desktop);
+  }
+}
 
-.electric-section-two-part-one-title {
-  text-align: center;
-  font-size: var(--title-size-mobile);
+/*****PART ONE******/
+
+.electric-section-two-part-one-container{
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-areas:
+    "electric-section-two-part-one-copy-container"
+    "electric-section-two-part-one-image-container";
+  row-gap: var(--subcontainer-row-gap-mobile);
+  column-gap: none;
+  /* ^for the media query after */
+  border: var(--subcontainer-border-mobile);
+  background-color: var(--level2-background-color);
+}
+
+@media (min-width:500px) {
+  .electric-section-two-part-one-container{
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-areas:
+    "electric-section-two-part-one-copy-container electric-section-two-part-one-image-container";
+    column-gap: var(--subcontainer-column-gap-desktop);
+    padding: var(--subcontainer-padding-desktop);
+  }
+}
+
+.electric-section-two-part-one-copy-container{
+  grid-area: electric-section-two-part-one-copy-container;
+  display: grid;
+  align-content: statrt;
+  row-gap: var(--copy-container-row-gap-mobile);
+  background-color: var(--level3-background-color);
+}
+
+@media (min-width: 500px) {
+  .electric-section-two-part-one-copy-container {
+    row-gap: var(--copy-container-row-gap-desktop);
+  }
 }
 
 .electric-section-two-part-one-subtitle {
   text-align: center;
-  font-size: var(--subtitle-size-mobile);
+  font-size: var(--subtitle-font-size-mobile);
+  font-weight: var(--subtitle-font-weight-mobile);
+}
+
+@media (min-width: 500px) {
+  .electric-section-two-part-one-subtitle {
+    text-align: left;
+    font-size: var(--subtitle-font-size-desktop);
+    font-weight: var(--subtitle-font-weight-desktop);
+  }
+}
+
+.electric-section-two-part-one-paragraph-container {
+  display: grid;
+  text-align: center;
+  row-gap: var(--copy-container-row-gap-mobile);
+  font-size: var(--paragraph-font-size-mobile);
+  font-weight: var(--paragraph-font-weight-mobile);
+}
+
+@media (min-width: 500px) {
+  .electric-section-two-part-one-paragraph-container{
+    text-align: left;
+    row-gap: var(--copy-container-row-gap-desktop);
+    font-size: var(--paragraph-font-size-desktop);
+    font-weight: var(--paragraph-font-weight-desktop);
+  }
 }
 
```

Comment:
there is no paragraph selector, can you add one and move the font-size font-weight in there, and make a media query

do that for all the part 1 2 3